### PR TITLE
[CloudAtCost.com] New ruleset

### PR DESCRIPTION
**Pull request by PeterDaveHello**

### CODE REVIEW

File: src/chrome/content/rules/CloudAtCost.com.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+        <test url="http://panel.cloudatcost.com/" />
+        <test url="http://panel.cloudatcost.com:12345/" />
+        <test url="http://panel.cloudatcost.com:59848/console.html" />
+
+    <rule from="^http:" to="https:" />
+
+        <test url="http://panel.cloudatcost.com/" />
+        <test url="http://members.cloudatcost.com/" />
+        <test url="http://panel.cloudatcost.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/CloudAtCost.com.xml]: They are already covered by the target hosts

File: src/chrome/content/rules/CloudAtCost.com.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            www.cloudatcost.com
+            forum.cloudatcost.com
+-->
+
+<ruleset name="cloudatcost.com">
+
+    <target host="panel.cloudatcost.com" />
+    <target host="members.cloudatcost.com" />
+
+    <exclusion pattern="^http://panel\.cloudatcost\.com:[0-9]+/" />
```
Comment (J0WI) [inline src/chrome/content/rules/CloudAtCost.com.xml]: You could also use ```\d``` for digits

**Comment by J0WI** [comment]
lgtm

**Comment by PeterDaveHello** [comment]
:+1:

**Pull request merged**